docker/elasticsearch/Dockerfile

```diff
@@ -1,2 +1,3 @@
 FROM blacktop/elasticsearch:6.3
 RUN bin/elasticsearch-plugin install --batch --silent ingest-attachment
+RUN apk add --no-cache curl
```

Comment:
Take you've pushed the latest image? We should split this out into its own repo so releases are autobuilt & published

Reply:
Yup pushed